doc/src/concepts/proxies.md

```diff
@@ -0,0 +1,29 @@
+# Proxies
+
+`rustup` provides a number of wrappers for common Rust tools.
+These are called _proxies_ and represent commands which are
+provided by the various [components][].
+
+The list of proxies is currently static in `rustup` and is as follows:
+
+[components]: concepts/components.html
+
+- `rustc`: `rustc` is the compiler for the Rust programming language, provided by the project itself and come the from the `rustc` component.
```

Comment:
This is still repeating the `rustc` bit.  Simpler would be:

```suggestion
- `rustc` is the compiler for the Rust programming language, provided by the project itself and come the from the `rustc` component.
```

Reply:
Similar simplifications can be done throughout and also will resolve some of the consistency issues below where you sometimes use backticks on command names and sometimes fail to.